lib/QubitInformationObjectXmlCache.class.php

```diff
@@ -84,27 +84,43 @@ public function export($resource)
   public function exportAll($options = array())
   {
     $skip = isset($options['skip']) ? $options['skip'] : 0;
-
-    // Get not-root and published information objects
-    $criteria = new Criteria;
-    $criteria->add(QubitInformationObject::ID, QubitInformationObject::ROOT_ID, Criteria::NOT_EQUAL);
-    $criteria = QubitAcl::addFilterDraftsCriteria($criteria);
-    $criteria->addAscendingOrderByColumn(QubitInformationObject::ID); // sort so optional skip will be consistent
-    $criteria->setOffset($skip);
-
-    $results = QubitInformationObject::get($criteria);
+    $limit = isset($options['limit']) ? $options['limit'] : 0;
```

Comment:
This is probably unnecessary since you're adding a default value to the option.

Reply:
Ah true... I'll remove.

Reply:
Oh, sorry, I just noticed it's on a different class, it may be a good idea to leave it just in case.

Reply:
Sounds good.